tools/hacky_postprocess_around_closure_limitations.py

```diff
@@ -48,6 +48,9 @@
 # var Module=function(a){
 f = re.sub(r'\s*function\s*\(Module\)\s*{\s*Module\s*=\s*Module\s*\|\|\s*{\s*}\s*;\s*var\s+(\w+)\s*=\s*Module\s*;', r'function(\1){', f)
 
+# Same as above but for arrow function
+f = re.sub(r'\s*\(Module\)\s*=>\s*{\s*Module\s*=\s*Module\s*\|\|\s*{\s*}\s*;\s*var\s+(\w+)\s*=\s*Module\s*;', r'(\1)=>{', f)
```

Comment:
Is there a test that exercises this?

Reply:
Yes, without there there is a test that fails... i don't remember which one I'm afraid.